Documentation/development-environment.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Multi-Node Test Environment
+weight: 91
+indent: true
+---
+
+# Rook development workflow
+
+## Setup expectation
+
+There are a bunch of pre-requisites to be able to deploy the following environment. Such as:
+
+* A Linux workstation (CentOS or Fedora)
+* KVM/QEMU installation
+
+For other Linux distribution, there is no guarantee the following will work.
+However adapting commands (apt/yum/dnf) could just work.
+
+## Prerequisites installation
+
+Execute `tests/scripts/multi-node/redhat-system-prerequisites.sh`.
```

Comment:
What about a more generic name for this such as `rpm-system-prerequisites.sh`?

Reply:
ok